### PR TITLE
add functions for build-container handling

### PR DESCRIPTION
**Pull request by ObiWahn**

### CODE REVIEW

File: helper.linux.fish [hunk 1]
```diff
@@ -52,6 +54,12 @@ end
 function pushCentosPackagingImage ; docker push $CENTOSPACKAGINGIMAGE ; end
 function pullCentosPackagingImage ; docker pull $CENTOSPACKAGINGIMAGE ; end
 
+function buildContainerDocumentation
```
Comment (fceller) [inline helper.linux.fish]: Why not "buildDocumentationImage"? see above

Comment (Simran-B) [reply helper.linux.fish]: The build script is guessed buildContainerDocumentation, I guess the function name was derived from that?

Comment (ObiWahn) [reply helper.linux.fish]: Yes - It was there long before integrating the call into the rest of oskar. I agree that it should be named equally.

File: helper.linux.fish [hunk 2]
```diff
@@ -61,6 +69,7 @@ function remakeImages
   pushUbuntuPackagingImage
   buildCentosPackagingImage
   pushCentosPackagingImage
+  buildDocumentationImage
 end
```
[the remark this answers is not in the document]
Comment (fceller) [reply helper.linux.fish]: Agreed. The script should also be split in much finer parts:
- update version
- generate examples & swagger
- build documents

**Pull request merged**